File: BlogBoutique/Models/BlogModel.cs
using BlogBoutique.Models;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text.Json.Serialization;

namespace BlogBoutique.Models
{
    [Table("Blog")]
    public class BlogModel
    {
        public virtual List<BlogBlogTypeModel>? BlogBlogTypes { get; set; } = new List<BlogBlogTypeModel>();
        [Key]
        public Int64 BlogId { get; set; }
        [ForeignKey("User")]
        public Int64 UserId { get; set; }
        public String? Title { get; set; } = null;
        public String? Text { get; set; } = null;
        public Int64? Upvote { get; set; } = null;
        public DateTime? DateCreated { get; set; } = null;
        public DateTime? DateModified { get; set; } = null;
    }
}

File: BlogBoutique/ClientApp/src/app/models/blog-model.ts
import { BlogBlogTypeModel } from "./blog-blog-type-model";

export class BlogModel {
  public selected?= false;

  public blogId: number = 0;
  public title?: string = '';
  public text?: string = '';
  public image?: string = '';
  public upvote?: number = 0;
  public userId?: number = 0;
  public dateCreated?: Date = undefined;
  public dateModified?: Date = undefined;

  public blogBlogTypes: BlogBlogTypeModel[] = [];

  constructor(init?: Partial<BlogModel>) {
    Object.assign(this, init);
  }
}
